core/src/main/kotlin/org/evomaster/core/problem/rest/resource/ResourceToken.kt

```diff
@@ -0,0 +1,78 @@
+package org.evomaster.core.problem.rest.resource
+
+open class RToken(
+    val originalText : String,
+    protected val lemma : String,
+    var assuredVerb : Boolean = false
```

Comment:
what is an `assuredVerb `?

Reply:
I have added comments for all properties. 

`assuredVerb`  indicates whether the token is a verb. In our case, the tokens in a path is not a complete sentence, so a word may be a noun or a verb. so I use `assuredVerb` to present that the word must be a verb.

Reply:
can u just add this comment? ie, the `assuredVerb indicates whether the token is a verb. In our case, the tokens in a path is not a complete sentence, so a word may be a noun or a verb. so I use assuredVerb to present that the word must be a verb.`
with the `I use` replaced by `we use`